circle.yml

```diff
@@ -243,13 +270,29 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - linux-gcc6
+      - linux-gcc6:
+          filters:
+            tags:
+              only: /.*/
+
       - deploy:
           requires:
             - macos-xcode90
             - linux-clang6
+            - linux-gcc6
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v.*/
```

Comment:
I don't know how it works exactly, but maybe this should be something more like `/v[0-9].*/` ?
For the possibility of creating a tag starting with `v` but not a version